### ISSUE TITLE
Provide OSX distribution for Mac

### ISSUE THREAD

**Issue by wwalc** [zadam/trilium#115]
It would be lovely to use CKEditor 5 for making local notes. I'm using OSX every day, so far I just tried it on Windows on a VM and it worked, so... :)

**Comment by zadam** [comment]
Hello, main reason for not providing MacOS build is that I don't have a way to test it. But I guess it will come sooner or later ...

**Comment by wwalc** [comment]
For anyone willing to give it a try on OSX:

```
./node_modules/.bin/electron-rebuild --arch=x64
./node_modules/.bin/electron-packager . --out=dist --platform=darwin --arch=x64 --overwrite
```

Kinda works 🎉

**Comment by zadam** [comment]
First mac build has been released as part of [0.25.0-beta](https://github.com/zadam/trilium/releases).
